harness/determined/cli/shell.py

```diff
@@ -36,46 +33,44 @@ def start_shell(args: Namespace) -> None:
     config = parse_config(args.config_file, None, args.config, args.volume)
     resp = launch_command(
         args.master,
-        "shells",
+        "api/v1/shells",
         config,
         args.template,
         context_path=args.context,
         data=data,
-    )
+    )["shell"]
+    print(resp)
```

Comment:
Not sure if this was used for debugging purposes, but it should be removed.

Reply:
good catch. removed it